DeusEx/Classes/DXRBannedItems.uc

```diff
@@ -76,6 +78,12 @@ function CheckConfig()
     }
 }
 
+function AnyEntry()
+{
+    Super.AnyEntry();
+    banneditems = dxr.flags.banneditems;
```

Comment:
Might also want to go through the player's inventory here to remove any banned items